e2e/content-services/document-list/document-list-pagination.e2e.ts

```diff
@@ -349,20 +349,28 @@ describe('Document List - Pagination', () => {
         await contentServicesPage.openFolder(newFolderModel.name);
         await expect(await contentServicesPage.getActiveBreadcrumb()).toEqual(newFolderModel.name);
         await expect(await paginationPage.getCurrentItemsPerPage()).toEqual(itemsPerPage.five);
-        await contentServicesPage.createAndOpenNewFolder(folderTwoModel.name);
+
+        await apiService.getInstance().login(acsUser.email, acsUser.password);
+        await contentServicesPage.createNewFolder(folderTwoModel.name);
+        const nodeIdSubFolderTwo = await contentServicesPage.getAttributeValueForElement(folderTwoModel.name, 'Node id');
+        await contentServicesPage.openFolder(folderTwoModel.name);
 
         for (let i = 0; i < numberOfSubFolders; i++) {
-            await contentServicesPage.createNewFolder('subfolder' + (i + 1));
+            await uploadActions.createFolder('subfolder' + (i + 1), nodeIdSubFolderTwo);
```

Comment:
Do we need to create the folders and subfolders in the test? i think they can be part of preconditions, in beforeAll

Reply:
I didn;'t change the test just the way it was doing , in this way the API are faster. We can refactor the test later